packages/cxl-ui/src/components/cxl-card.js

```diff
@@ -11,8 +12,85 @@ export class CXLCardElement extends LitElement {
     return [cxlCardStyles];
   }
 
+  separator = html`<span> | </span>`;
+
+  @state() _hasSlottedHeaderTags = false;
+
+  @property({ type: String }) id = '';
+
+  @property({ type: String }) type = 'Course';
+
+  @property({ type: String }) title = '';
+
+  @property({ type: String }) time = '';
+
+  @property({ type: String }) instructor = '';
+
+  @property({ type: String, attribute: 'avatar-image' }) avatarImage = '';
+
+  @property({ type: Boolean, reflect: true }) new = false;
```

Comment:
@freudFlintstone you mentioned that @kertuilves used different approach for this state of the card. This looks good at the first glance but what do you think about introducing `status` attribute and property which would be more universal? It could have `new`, `completed` etc.

Reply:
Sure, sounds good!